### ISSUE TITLE
Change go2 training terrain

### ISSUE THREAD

**Issue by Jihy** [unitreerobotics/unitree_rl_gym#52]
I noticed that the provided code only has a ground terrain to train the go2. So I added other terrain like the stairs, uniform terrain which is provided by issacgym (trimesh), but it failed when training with the same reward. However, some of the terrain is so eazy that even go2 trained on provided ground can cover the task, the training curve failed and go2 can't walk.
I'm wondering if this is related to the trimesh property in sim or some other bugs.
p.s. To complatedly exclude the terrain difficulty, I even create a trimesh ground, but it also failed.

The trimesh code is the same way as ground: 
`    def _create_trimesh(self):
        """ Adds a triangle mesh terrain to the simulation, sets parameters based on the cfg.
        # """
        tm_params = gymapi.TriangleMeshParams()
        tm_params.nb_vertices = self.terrain.vertices.shape[0]
        tm_params.nb_triangles = self.terrain.triangles.shape[0]

        tm_params.transform.p.x = -self.terrain.cfg.border_size 
        tm_params.transform.p.y = -self.terrain.cfg.border_size
        tm_params.transform.p.z = 0.0
        tm_params.static_friction = self.cfg.terrain.static_friction
        tm_params.dynamic_friction = self.cfg.terrain.dynamic_friction
        tm_params.restitution = self.cfg.terrain.restitution
        print("vertices: ", self.terrain.vertices.flatten(order='C').shape)
        print("triangles: ", self.terrain.triangles.flatten(order='C').shape)
        self.gym.add_triangle_mesh(self.sim, self.terrain.vertices.flatten(order='C'), self.terrain.triangles.flatten(order='C'), tm_params)   
        self.height_samples = torch.tensor(self.terrain.heightsamples).view(self.terrain.tot_rows, self.terrain.tot_cols).to(self.device)`

**Comment by frro-0107** [comment]
想问一下直接拿这个官方仓库的训练GO2的代码一点不改动 能训练成功走路吗？我用了python legged_gym/scripts/train.py --task=go2 --num_envs=513 --max_iterations=30000 训练出来的效果是狗头杵在地上一动不动，向您请教一下 I would like to ask you directly to this official warehouse training GO2 code without changing, can you train to successfully walk? I trained with‘ python legged_gym/scripts/train.py --task=go2 --num_envs=512 --max_iterations=30000‘The result  is that the dog's head is motionless on the ground, .

**Comment by kuainx** [comment]
> 想问一下直接拿这个官方仓库的训练GO2的代码一点不改动 能训练成功走路吗？我用了python legged_gym/scripts/train.py --task=go2 --num_envs=513 --max_iterations=30000 训练出来的效果是狗头杵在地上一动不动，向您请教一下 I would like to ask you directly to this official warehouse training GO2 code without changing, can you train to successfully walk? I trained with‘ python legged_gym/scripts/train.py --task=go2 --num_envs=512 --max_iterations=30000‘The result is that the dog's head is motionless on the ground, .

我遇到了相同的问题
